packages/trader/src/AppV2/Containers/Trade/trade.tsx

```diff
@@ -76,6 +86,19 @@ const Trade = observer(() => {
         }
     }, []);
 
+    React.useEffect(() => {
+        if (!default_stake || prev_contract_type === contract_type) return;
```

Comment:
`prev_contract_type === contract_type` check stands here for reloading page. After reloading `prev_contract_type === contract_type` ---> `true`, so we won't insert `default_stake` and save what used have typed.